CHANGELOG_PENDING.md

```diff
@@ -20,13 +30,19 @@ Friendly reminder, we have a [bug bounty program](https://hackerone.com/tendermi
 
 ### FEATURES
 
-- [#9083] backport cli command to reindex missed events (@cmwaters)
+- [cli] \#9083 Backport command to reindex missed events (@cmwaters)
 - [cli] \#9107 Add the `p2p.external-address` argument to set the node P2P external address (@amimart)
 
 ### IMPROVEMENTS
 
-- [config] \#9054 Flag added to overwrite abciresponses.
+- [config] \#9054 `discard_abci_responses` flag added to discard all ABCI
+  responses except the last in order to save on storage space in the state
+  store (@samricotta)
+- [config] \#9275 Move the `discard_abci_responses` flag into a new `[storage]`
+  section.
```

Comment:
nit: Not sure if this is necessary. Users will never notice that it was originally in another section.

Reply:
Cool, I've removed it.